ts/test/test-init-config.ts

```diff
@@ -42,6 +43,16 @@ describe('initConfig', () => {
     process.env = savedEnv;
   });
 
+  const internalConfigParams = {
+    timeIntervalMicros: 1000,
+    heapIntervalBytes: 512 * 1024,
+    heapMaxStackDepth: 64,
+    initialBackoffMillis: 1000,
+    maxBackoffMillis: 60 * 60 * 1000,
+    backoffMultiplier: 1.3,
+    backoffLimitMillis: 7 * 24 * 60 * 60 * 1000
```

Comment:
Use parseDuration here and for other fields?

Reply:
I want to avoid using it here parseDuration in tests for initConfig to kind of confirm that parseDuration is working as expected.